src/main/kotlin/no/nav/syfo/service/KartleggingssporsmalVarselService.kt

```diff
@@ -0,0 +1,92 @@
+package no.nav.syfo.service
+
+import no.nav.syfo.BRUKERNOTIFIKASJONER_MER_VEILEDNING_MESSAGE_TEXT
+import no.nav.syfo.Environment
+import no.nav.syfo.MER_VEILEDNING_URL
+import no.nav.syfo.kafka.consumers.varselbus.domain.ArbeidstakerHendelse
+import no.nav.syfo.kafka.producers.dittsykefravaer.domain.DittSykefravaerMelding
+import no.nav.syfo.kafka.producers.dittsykefravaer.domain.DittSykefravaerVarsel
+import no.nav.syfo.kafka.producers.dittsykefravaer.domain.OpprettMelding
+import no.nav.syfo.kafka.producers.dittsykefravaer.domain.Variant
+import no.nav.syfo.metrics.countKartleggingssporsmalVarselSendt
+import no.nav.syfo.service.SenderFacade.InternalBrukernotifikasjonType.OPPGAVE
+import java.net.URI
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+import java.util.UUID
+
+class KartleggingssporsmalVarselService(
+    val senderFacade: SenderFacade,
+    val env: Environment,
+    val accessControlService: AccessControlService,
+) {
+    suspend fun sendKartleggingssporsmalTilArbeidstaker(
+        arbeidstakerHendelse: ArbeidstakerHendelse,
+    ) {
+        val userAccessStatus = accessControlService.getUserAccessStatus(arbeidstakerHendelse.arbeidstakerFnr)
+        if (userAccessStatus.canUserBeDigitallyNotified) {
```

Comment:
Det er intensjonen at om arbeidstaker har reservert seg mot å kontaktes digitalt, så skal man ikke falle tilbake på fysisk brev for disse kartleggingsspørsmålene?

Reply:
Usikker. Kan bli vanskelig å svare på fysisk. Har vi noe sted de eventuelt vil kunne svare? Vil vi åpne skjemaet digitalt, og sende brevet fysisk, så vil de kunne gå inn digitalt for å fylle det ut?